### ISSUE TITLE
Move Deep Blue models into DeepBlue namespace

### ISSUE THREAD

**Issue by respinos** [mlibrary/hydra-prototype#30]
```
bundle exec rails g curation_concerns:work Item
# bundle exec rails g curation_concerns:work Collection
```

(Just the Item.)

**Comment by respinos** [comment]
Guess before we load too much data into the repository!

**Comment by njaffer** [comment]
Does that mean that Deep Blue Collection would be a Generic Work and not Collection?

**Comment by respinos** [comment]
Good catch: it would generate it as a GenericWork, but we'd just swap out the includes.

BUT the generator is broken with respect to namespaces, so this isn't going to work. Investigating.

**Comment by respinos** [comment]
Curation Concerns clearly does not expect you to namespace your models --- note that the Rackham controller is actually CurationConcerns::RackhamsController. I don't know if we want CurationConcerns::DeepBlue::ItemController, but I need to do further research.

I'm also baffled why Curation Concerns doesn't configure rspec tests for the collection objects.

**Comment by respinos** [comment]
This explains my motivation for namespaces: 

http://blog.makandra.com/2014/12/organizing-large-rails-projects-with-namespaces/

It's easy to see we'd end up with gems for "umich-deep-blue" and "umich-dlps-image" and "umich-dlps-findaid", any combination of being used in a head.

But I do suspect the namespacing is beyond the scope of this prototype.

**Comment by njaffer** [comment]
Yep agreed. This is not the scope of this sprint

**Comment by blancoj** [comment]
Great link roger. Easy to understand namespacing now.  Thank you!
